tests/pom.xml

```diff
@@ -0,0 +1,52 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.jenkins.jenkinsfile-runner</groupId>
+    <artifactId>parent</artifactId>
+    <version>1.0-beta-5-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>tests</artifactId>
+
+  <dependencies>
+    <dependency>
```

Comment:
Is this dependency necessary?

Reply:
No. I forgot to remove it when I remove the test classes